### DISCUSSION TITLE
I2SPlay +/file.mp3 not working

### DISCUSSION THREAD

**Discussion by Slomo17** [arendst/Tasmota#21926] [Q&A] [answered]
Hey, i am trying to play a local mp3 file with tasmota. When i use`I2SPlay file.mp3` the file plays, but according to the manual it blocks the system until the sound is finished playing. As i want to simultaneously display some text on a display i tried using the  `I2SPlay +/file.mp3` (i also tried `I2SPlay +file.mp3`)  command as written [here](https://tasmota.github.io/docs/I2S-Audio/#commands), but this doesn't work, because tasmota can't finde the file `{"I2SPlay":"File not found"}`. Am i doing something wrong or is this command deprecated, or did i miss a setting when compiling with audio support?

Thanks in advance for any help.

**Comment by Staars** [comment, marked as answer]
The new behavior is always non blocking.
So the „+“ is not supported anymore.

If you have a use case, where you need the old behavior, this feature could be reintroduced in one or the other way.

**Reply by sfromis** [reply]
Edited the docs, to remove the `+` and blocking from the I2SPlay command description.

**Comment by Slomo17** [comment]
Ah ok, good to know, i thought it was blocking because when i tried to trigger both the sound an the display via mqtt i noticed a delay with the display, i will test it again. Thanks for the information.

**Reply by Jason2866** [reply]
Non blocking, but decoding mp3 is taking MCU power, so there will be a performance slow down for other tasks.

**Reply by Slomo17** [reply]
ok that's what i guessed. Will an Rtttl file/string be less performance hungry?  i only need a small short acoustic feedback.

**Reply by sfromis** [reply]
I'd suggest to just try it, and see what happens.

**Reply by Slomo17** [reply]
I will do that, thanks
